docs/config-definition.md

```diff
@@ -38,7 +41,17 @@ scheduleSyncPeriod: 1m
 restoreOnlyMode: false
 ```
 
-## Parameter Reference
+## Install in any namespace
+
+To install Ark in another namespace, you can specify the namespace in your `Config` YAML. That is, you can replace the value `heptio-ark` in the example with the appropriate namespace value.
+
+Or you can run:
```

Comment:
This is **only** for the CLI. This is complementary. If you install the ark server into the `foo` namespace, you would likely want to run `ark client config set namespace=foo` so that any time you run `ark backup create` or any of the other CLI commands, you avoid needing to specify `--namespace=foo`.

Reply:
At a minimum, will clarify. See also previous comment.